### PR TITLE
Adding IDOs/phase 2

### PR DESCRIPTION
**Pull request by Mttbnchtt**
I am going to add all the remaining 12 IDOs to the class "intermediate development outcome".

### CODE REVIEW

File: src/sdgio-edit.owl [hunk 1]
```diff
@@ -2766,6 +2778,101 @@ AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <ht
 AnnotationAssertion(rdfs:label <http://purl.unep.org/sdg/SDGIO_00020239> "enhanced smallholder market access"@en)
 ClassAssertion(<http://purl.unep.org/sdg/SDGIO_00020236> <http://purl.unep.org/sdg/SDGIO_00020239>)
 
+# Individual: <http://purl.unep.org/sdg/SDGIO_00020240> (Increased incomes and employment)
```
Comment (pbuttigieg) [inline src/sdgio-edit.owl]: I'd say this can be lowercase, it's not a full sentence.

File: src/sdgio-edit.owl [hunk 2]
```diff
@@ -2766,6 +2778,101 @@ AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <ht
...
+
+# Individual: <http://purl.unep.org/sdg/SDGIO_00020243> (improved food security)
+
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.unep.org/sdg/SDGIO_00020243> <http://orcid.org/0000-0062-3253-4387>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> <http://purl.unep.org/sdg/SDGIO_00020243> "2019-04-29T16:27:28Z"^^xsd:dateTime)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <http://purl.unep.org/sdg/SDGIO_00020243> <https://clarisa.cgiar.org/api/srf-idos/6>)
+AnnotationAssertion(rdfs:label <http://purl.unep.org/sdg/SDGIO_00020243> "improved food security"@en)
+ClassAssertion(<http://purl.unep.org/sdg/SDGIO_00020236> <http://purl.unep.org/sdg/SDGIO_00020243>)
+
+# Individual: <http://purl.unep.org/sdg/SDGIO_00020244> (improved human and animal health through better agricultural practices)
```
Comment (pbuttigieg) [inline src/sdgio-edit.owl]: Nothing to resolve, just a comment to think about in the next phase: 
"improved" is a really slippery idea - we should ask the strategy developers what it means and how they judge it in their context.

File: src/sdgio-edit.owl [hunk 3]
```diff
@@ -2766,6 +2778,101 @@ AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <ht
...
+
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.unep.org/sdg/SDGIO_00020244> <http://orcid.org/0000-0062-3253-4387>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> <http://purl.unep.org/sdg/SDGIO_00020244> "2019-04-29T16:28:08Z"^^xsd:dateTime)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <http://purl.unep.org/sdg/SDGIO_00020244> <https://clarisa.cgiar.org/api/srf-idos/7>)
+AnnotationAssertion(rdfs:label <http://purl.unep.org/sdg/SDGIO_00020244> "improved human and animal health through better agricultural practices"@en)
+ClassAssertion(<http://purl.unep.org/sdg/SDGIO_00020236> <http://purl.unep.org/sdg/SDGIO_00020244>)
+
+# Individual: <http://purl.unep.org/sdg/SDGIO_00020245> (natural capital enhanced and protected, especially from climate change)
```
Comment (pbuttigieg) [inline src/sdgio-edit.owl]: As with "improved", "enhanced" is a slipper idea. We should get information about what that means too and add it as a comment on this instance (in a separate PR)

File: src/sdgio-edit.owl [hunk 4]
```diff
@@ -2766,6 +2778,101 @@ AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <ht
...
+
+# Individual: <http://purl.unep.org/sdg/SDGIO_00020246> (enhanced benefits from ecosystem goods and services)
+
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.unep.org/sdg/SDGIO_00020246> <http://orcid.org/0000-0062-3253-4387>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> <http://purl.unep.org/sdg/SDGIO_00020246> "2019-04-29T16:29:41Z"^^xsd:dateTime)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <http://purl.unep.org/sdg/SDGIO_00020246> <https://clarisa.cgiar.org/api/srf-idos/9>)
+AnnotationAssertion(rdfs:label <http://purl.unep.org/sdg/SDGIO_00020246> "enhanced benefits from ecosystem goods and services"@en)
+ClassAssertion(<http://purl.unep.org/sdg/SDGIO_00020236> <http://purl.unep.org/sdg/SDGIO_00020246>)
+
+# Individual: <http://purl.unep.org/sdg/SDGIO_00020247> (more sustainably managed agro-ecosystems)
```
Comment (pbuttigieg) [inline src/sdgio-edit.owl]: We'll also need to get the CGIAR's criteria for what makes something "sustainably" managed.

File: src/sdgio-edit.owl [hunk 5]
```diff
@@ -2766,6 +2778,101 @@ AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <ht
...
+
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.unep.org/sdg/SDGIO_00020247> <http://orcid.org/0000-0062-3253-4387>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> <http://purl.unep.org/sdg/SDGIO_00020247> "2019-04-29T16:30:19Z"^^xsd:dateTime)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <http://purl.unep.org/sdg/SDGIO_00020247> <https://clarisa.cgiar.org/api/srf-idos/10>)
+AnnotationAssertion(rdfs:label <http://purl.unep.org/sdg/SDGIO_00020247> "more sustainably managed agro-ecosystems"@en)
+ClassAssertion(<http://purl.unep.org/sdg/SDGIO_00020236> <http://purl.unep.org/sdg/SDGIO_00020247>)
+
+# Individual: <http://purl.unep.org/sdg/SDGIO_00020248> (mitigation and adaptation achieved)
```
Comment (pbuttigieg) [inline src/sdgio-edit.owl]: We'll need to ask for what the CGIAR criteria for the achievement of this and similarly worded objective are.

File: src/sdgio-edit.owl [hunk 6]
```diff
@@ -2766,6 +2778,101 @@ AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <ht
...
+
+# Individual: <http://purl.unep.org/sdg/SDGIO_00020249> (equity and inclusion achieved)
+
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.unep.org/sdg/SDGIO_00020249> <http://orcid.org/0000-0062-3253-4387>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> <http://purl.unep.org/sdg/SDGIO_00020249> "2019-04-29T16:31:30Z"^^xsd:dateTime)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <http://purl.unep.org/sdg/SDGIO_00020249> <https://clarisa.cgiar.org/api/srf-idos/B1>)
+AnnotationAssertion(rdfs:label <http://purl.unep.org/sdg/SDGIO_00020249> "equity and inclusion achieved"@en)
+ClassAssertion(<http://purl.unep.org/sdg/SDGIO_00020236> <http://purl.unep.org/sdg/SDGIO_00020249>)
+
+# Individual: <http://purl.unep.org/sdg/SDGIO_00020250> (enabling environment improved)
```
Comment (pbuttigieg) [inline src/sdgio-edit.owl]: We need a class in SDGIO for "enabling environment" - which I suspect will be difficult to define.

Comment (marieALaporte) [reply src/sdgio-edit.owl]: @pbuttigieg Shouldn't at least the instance be defined as an instance of the IDO class?

Comment (pbuttigieg) [reply src/sdgio-edit.owl]: @marieALaporte Yes, there should be a "Types" assertion that this is an IDO.

@Mttbnchtt your protege should look like this:

image

Comment (Mttbnchtt) [reply src/sdgio-edit.owl]: @marieALaporte noted that the instance SDGIO_00020250 did not have. I added the instance to the class "intermediate development outcome".

@pbuttigieg: the class that you showed in the picture looks good, I suppose. Is there a problem with that one?

Comment (marieALaporte) [reply src/sdgio-edit.owl]: @Mttbnchtt everything looks good. Thanks.

File: src/sdgio-edit.owl (hunk 4 above)
Comment (Mttbnchtt) [reply src/sdgio-edit.owl]: I added issue #149.

File: src/sdgio-edit.owl (hunk 3 above)
Comment (Mttbnchtt) [reply src/sdgio-edit.owl]: I added issue #148.

File: src/sdgio-edit.owl (hunk 2 above)
Comment (Mttbnchtt) [reply src/sdgio-edit.owl]: I added issue #147.

File: src/sdgio-edit.owl (hunk 6 above)
Comment (pbuttigieg) [reply src/sdgio-edit.owl]: > @pbuttigieg: the class that you showed in the picture looks good, I suppose. Is there a problem with that one?

I added that quickly for visualisation but didn't commit it. You've fixed this now.

**Comment by pbuttigieg** [comment]
Thanks @marieALaporte !
Merging now...

File: src/sdgio-edit.owl (hunk 5 above)
Comment (Mttbnchtt) [reply src/sdgio-edit.owl]: I added issue #150.

File: src/sdgio-edit.owl (hunk 6 above)
Comment (Mttbnchtt) [reply src/sdgio-edit.owl]: @pbuttigieg: Do I need to define a class "enabling environment" now or should I rather create an issue?

File: src/sdgio-edit.owl (hunk 1 above)
Comment (Mttbnchtt) [reply src/sdgio-edit.owl]: @pbuttigieg I have not changed this yet. I will do it in the next PR (which will mainly be about sub-IDOs).

**Pull request merged**